recipes/gmtl/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "gmtl"
+    description = "Generic Math Template Library"
```

Comment:
You might expand in the description a bit more instead of just using the name.

```suggestion
    The Generic Math Template Library. A math library designed to be high-performance, extensible, and generic.
```